.env.example

```diff
@@ -0,0 +1,4 @@
+CCAO_REGISTRY_URL=ghcr.io/ccao-data
+AWS_S3_MODEL_BUCKET=s3://ccao-model-results-us-east-1
+AWS_S3_DEFAULT_MODEL_RUN_ID=2025-02-11-charming-eric
+API_PORT=3636
```

Comment:
This change isn't strictly necessary as part of this PR, but I included it because it's helpful for local development.